packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -338,6 +340,13 @@ export class AtomicSearchInterface
     this.engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
   }
 
+  @Method() public async getOrganizationEndpoints(
+    organizationId: string,
+    env: PlatformEnvironment = 'prod'
+  ) {
+    return getOrganizationEndpointsHeadless(organizationId, env);
+  }
```

Comment:
This utility neither reads something from the search interface nor writes anything to it, so I'm not sure if we should expose it through the search interface like this.

Instead, perhaps we should add it to the global `CoveoAtomic` object and our `atomic/src/index.ts` file. We could even decide to re-export everything from `atomic/src/index.ts` in the global `CoveoAtomic` object.

I'm overall hesitant though, since we don't usually direct people towards using the global object.